log.go

```diff
@@ -37,6 +38,7 @@ func UseLogger(logger btclog.Logger) {
 	addrmgr.UseLogger(logger)
 	blockntfns.UseLogger(logger)
 	pushtx.UseLogger(logger)
+	connmgr.UseLogger(logger)
```

Comment:
FWIW, when it's run within `lnd`, we've already hooked up the `connmgr` as a logger.